src/utils/management/commands/install_janeway.py

```diff
@@ -38,7 +40,7 @@ def handle(self, *args, **options):
         :return: None
         """
         call_command('migrate')
-        print("Please answer the following questions.\n")
+        logger.info("Please answer the following questions.\n")
```

Comment:
I think using print here (and in the ones below) was the right thing to do since this is a CLI program and we expect the user to read and react to these